front/src/styles/scss/common/components/_typeAndPath.scss

```diff
@@ -1,15 +1,14 @@
 .type-and-path {
   position: relative;
+
   background-color: var(--white);
   padding: 0.5rem;
   border-radius: var(--border-radius);
   .help {
     position: relative;
-    display: flex;
     align-items: center;
     justify-content: center;
     height: 5rem;
-    max-width: 21vw;
```

Comment:
Like most of your changes in this PR, they weren't asked on the issue. This component is also used in the manage train schedule module in the itinerary tab so when you change the style here, it affects also this part of the app. 

I think before changing things like this, you should check with our ux, @thibautsailly, what he thinks about that

Reply:
I think you should leave the `display: flex` to benefit from the `align-items: center` which feels right.

The max-width needs also to stay otherwise the component grows too much in manage train schedule module. To have more space for the text, you could go to `max-width: 29vw` but I think @nicolaswurtz did it on purpose so you should leave it with 21vw.

The `justify-content: center` doesn't do anything so that's the only thing you can remove.

Reply:
ok updated